### ISSUE TITLE
The name org.freedesktop.secrets was not provided by any .service files

### ISSUE THREAD

**Issue by Klikini** [microsoft/vscode-docker#1515]
Repro steps:

1. Select "Connect registries..."
2. Enter username and password

Action: vscode-docker.registries.connectRegistry
Error type: Error
Error Message: The name org.freedesktop.secrets was not provided by any .service files

Version: 0.9.0
OS: linux (OpenSUSE Tumbleweed)
Product: Visual Studio Code
Product Version: 1.41.1
Language: en

**Comment by philliphoff** [comment]
@Klikini Is this error repeatable (on this machine, or others)?  This error can suggest that Linux credentials management components, such as `gnome-keyring` are not completely installed/configured/running.

**Comment by Klikini** [comment]
It is reproducible on this machine every time. I'm running KDE, so maybe it
should be kwallet instead of gnome-keyring?

**Comment by philliphoff** [comment]
The underlying problem seems to be that `kwallet` does not support the common API required by the Linux `libsecret` library, which is used by the `node-keytar` library on Linux as the generalized credential store.  So, on KDE, `gnome-keyring` is still required.  (This seems to have been the case for some time; see [this VS Code issue](https://github.com/MicrosoftDocs/live-share/issues/205) for links to more discussion.)

**Comment by Klikini** [comment]
Okay, I understand. I installed `gnome-keyring` and it works. So this is a duplicate of #205.

**Comment by ItalyPaleAle** [comment]
Stumbled upon this ticket while researching a similar issue.

Testing VS Code on a Chromebook with Crostini, after logging into GitHub I got this error:

![Screenshot 2020-11-18 at 19 19 00](https://user-images.githubusercontent.com/43508/99617602-810f4800-29d4-11eb-9c06-6a1850b5899f.png)

Following this issue, I fixed it by installing gnome-keyring.

However, I wonder if it would make sense to add gnome-keyring as a dependency for VS Code's package?
